docs/openapi.json

```diff
@@ -931,6 +931,9 @@
                     "modified_date": {
                         "type": "string"
                     },
+                    "packages": {
+                        "type": "string"
```

Comment:
This is not correct, we need to either create custom wrapper for JSONB, or use the actual AvisoryPackageDat struct here.

Reply:
fixed, also fixed CVEs format